Loaded dependency [python3/3.10.12]: gcc/11.4.0-binutils-2.40
Loaded dependency [python3/3.10.12]: sqlite3/3.42.0

Switching from python3/3.8.2 to python3/3.10.12
  Loading requirement: gcc/11.4.0-binutils-2.40 sqlite3/3.42.0
2024-02-05 19:26:44.303154: E tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc:9342] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-02-05 19:26:44.303223: E tensorflow/compiler/xla/stream_executor/cuda/cuda_fft.cc:609] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-02-05 19:26:44.348096: E tensorflow/compiler/xla/stream_executor/cuda/cuda_blas.cc:1518] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
/zhome/77/8/118225/.local/lib/python3.10/site-packages/numpy/core/getlimits.py:518: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/zhome/77/8/118225/.local/lib/python3.10/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
Traceback (most recent call last):
  File "/zhome/77/8/118225/Desktop/Projects/score_diffusion_mean/score_diffusion_mean/train_score.py", line 545, in <module>
    train_score()
  File "/zhome/77/8/118225/Desktop/Projects/score_diffusion_mean/score_diffusion_mean/train_score.py", line 475, in train_score
    train_s2(M=M,
  File "/zhome/77/8/118225/Desktop/Projects/score_diffusion_mean/score_diffusion_mean/jaxgeometry/statistics/score_matching/trainxt.py", line 320, in train_s2
    np.save(os.path.join(save_path, "loss_arrays.npy"), jnp.stack(loss))
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/numpy/lax_numpy.py", line 1797, in stack
    new_arrays.append(expand_dims(a, axis))
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/numpy/lax_numpy.py", line 903, in expand_dims
    return lax.expand_dims(a, axis)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/lax/lax.py", line 1336, in expand_dims
    return broadcast_in_dim(array, result_shape, broadcast_dims)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/lax/lax.py", line 803, in broadcast_in_dim
    return broadcast_in_dim_p.bind(
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/core.py", line 385, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/core.py", line 388, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/core.py", line 868, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/dispatch.py", line 140, in apply_primitive
    return compiled_fun(*args)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/dispatch.py", line 172, in <lambda>
    return lambda *args, **kw: call(*args, **kw)[0]
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/profiler.py", line 340, in wrapper
    return func(*args, **kwargs)
  File "/zhome/77/8/118225/.local/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 1152, in __call__
    results = self.xla_executable.execute_sharded(input_bufs)
KeyboardInterrupt
terminate called without an active exception
/zhome/77/8/118225/.lsbatch/1707157585.20016805.shell: line 37: 26577 Aborted                 python3 train_score.py --manifold SN --dim 2 --loss_type dsm --load_model 0 --T_sample 0 --t 0.01 --train_net s2 --max_T 1.0 --lr_rate 0.001 --epochs 50000 --x_samples 64 --t_samples 256 --repeats 16 --samples_per_batch 32 --dt_steps 1000 --save_step 10 --seed 2712
